gradle.properties

```diff
@@ -29,3 +29,6 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=twyatt
 POM_DEVELOPER_NAME=Travis Wyatt
 POM_DEVELOPER_URL=https://github.com/twyatt
+
+kotlin.js.webpack.major.version=5
+kotlin.js.compiler=both
```

Comment:
do we care about supporting both legacy and ir? @twyatt @cedrickcooke

Reply:
If it isn't a big lift, then I think we'll want to support legacy for this library. If it's causes you trouble though, I'm also fine with going with IR only.

Reply:
sounds good! I will leave it as then and support `Legacy` and `IR`